Need to compile helix-core tests for admin webapp

helix admin webapp depends on helix-core's test jar. So tests need to be compiled, as well. `-Dmaven.test.skip.exec=true` is needed.


```
[INFO] Apache Helix :: Core ............................... SUCCESS [ 18.881 s]
[INFO] Apache Helix :: Admin Webapp ....................... FAILURE [  0.092 s]
[INFO] Apache Helix :: Restful Interface .................. SKIPPED
[INFO] Apache Helix :: Distributed Lock ................... SKIPPED
[INFO] Apache Helix :: HelixAgent ......................... SKIPPED
[INFO] Apache Helix :: Recipes ............................ SKIPPED
[INFO] Apache Helix :: Recipes :: Rabbitmq Consumer Group . SKIPPED
[INFO] Apache Helix :: Recipes :: Rsync Replicated File Store SKIPPED
[INFO] Apache Helix :: Recipes :: distributed lock manager  SKIPPED
[INFO] Apache Helix :: Recipes :: distributed task execution SKIPPED
[INFO] Apache Helix :: Recipes :: service discovery ....... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  31.429 s
[INFO] Finished at: 2020-09-03T15:57:52-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project helix-admin-webapp: Could not resolve dependencies for project org.apache.helix:helix-admin-webapp:bundle:1.0.2-SNAPSHOT: Failure to find org.apache.helix:helix-core:jar:tests:1.0.2-SNAPSHOT in http://repository.apache.org/snapshots was cached in the local repository, resolution will not be reattempted until the update interval of apache.snapshots has elapsed or updates are forced -> [Help 1]
```

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@ Mailing list: http://helix.apache.org/mail-lists.html
 ### Build
 
 ```bash
-mvn clean install -Dmaven.test.skip=true
+mvn clean install -Dmaven.test.skip.exec=true
 ```
 
 ## WHAT IS HELIX

--- a/build
+++ b/build
@@ -19,4 +19,4 @@
 # under the License.
 #
 
-mvn clean install -Dmaven.test.skip=true
+mvn clean install -Dmaven.test.skip.exec=true

